code/src/main/java/frc/robot/common/TraceableCommand.java

```diff
@@ -45,7 +45,7 @@ public void end(boolean interrupted) {
   // Returns true when the command should end.
   @Override
   public boolean isFinished() {
-    return false;
+    return true;
```

Comment:
I'm guessing it may have been a bug, which came up in mocking the commands.  Usually it was never a problem for us because we always overwrite the method.

Reply:
Yeah by default we'd want commands to complete immediately. If you want it to keep going it really should be a conscience decision - as you're likely going to need logic to finally end it.